apps/cli/src/node/docker-compose-explorer.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.9"
+
+services:
+    database_creator:
+        image: postgres:13-alpine
+        command: ["createdb", "squid"]
+        environment:
+            PGHOST: database
+            PGPORT: 5432
+            PGUSER: postgres
+            PGPASSWORD: password
+            PGDATABASE: postgres
+        depends_on:
+            database:
+                condition: service_healthy
+
+    squid_api:
+        image: cartesi/rollups-explorer-api:0.1.0-rc.0
```

Comment:
IF we can have a variable here, it will ake it easy when updating.

Reply:
I don't think this is something we have to allow sunodo users to control.
Sunodo developers can just edit this file on their local dev environment.